### PR TITLE
Fix `The DEFAULT_FILE_STORAGE setting is deprecated` warning

### LINKED ISSUE #366: Provide a fix to avoid " The DEFAULT_FILE_STORAGE setting is deprecated"
Hello, thanks for a great app!

I used django-health-check for a while with no problems.
But recently I faced with the issue when I tried to execute a command `poetry run pytest tests`: 
django.utils.deprecation.RemovedInDjango51Warning: The DEFAULT_FILE_STORAGE setting is deprecated. Use STORAGES instead.
note: Django 4.2 introduces the new [STORAGES setting](https://docs.djangoproject.com/en/4.2/releases/4.2/#custom-file-storages).
Could you fix this issue

### PR DESCRIPTION
**Pull request by KOliver94**
Split from: https://github.com/revsys/django-health-check/pull/397

There are already two PRs for this (https://github.com/revsys/django-health-check/pull/359, https://github.com/revsys/django-health-check/pull/383) but as they are conflicting with master right now I included it here to avoid the anoying warnings.

Closes https://github.com/revsys/django-health-check/pull/359, closes https://github.com/revsys/django-health-check/pull/383, fixes https://github.com/revsys/django-health-check/issues/366

### CODE REVIEW

File: health_check/storage/backends.py [hunk 1]
```diff
@@ -81,4 +81,4 @@ def check_status(self, subset=None):
 
 class DefaultFileStorageHealthCheck(StorageHealthCheck):
     storage_alias = "default"
-    storage = settings.DEFAULT_FILE_STORAGE
+    storage = default_storage
```
Comment (50-Course) [inline health_check/storage/backends.py:84]: 👍🏼

**Review by 50-Course — APPROVED** [review]
Hello @KOliver94,

Foremost, thank you for your MR. I so happen to run into this same issue while maintaining `django-silk` earlier this year, making a pass at this. I have gone through the mentioned tickets (#359 & #383), as well as cross-referencing this and the blocking MRs. 

That said, this is good to go 🚀

**Comment by KOliver94** [comment]
Hey @frankwiles,

Is there a chance to check my PR and get rid of the warning?

Thanks!

**Comment by KOliver94** [comment]
Hey @frankwiles,

Django 5.1 is expected to be released in 3 months. The issue has been open for more than a year. There are 3 available PRs with fixes. Can you check #359, #383 or mine here please?

Thanks!

**Review by sobolevn — APPROVED** [review]
@codingjoe I am really troubled with this warning, because I have `-Werror` set. Everytime I access this in tests / development I see an error.

New release would be just amazing 🙏

**Review by codingjoe — APPROVED** [review]
I stepped down as a maintainer, but sure. This looks fine. Let's get it out the door.

**Pull request merged**